adafruit_portalbase/__init__.py

```diff
@@ -28,7 +28,6 @@
 from adafruit_bitmap_font import bitmap_font
 from adafruit_display_text import wrap_text_to_lines
 from adafruit_display_text.bitmap_label import Label
-from adafruit_display_text.outlined_label import OutlinedLabel
```

Comment:
I think we should wait a bit on changing this. Some of the device specific libraries that are use PortalBase are for devices with Display Text frozen in. 

Unless users of those devices put the latest Display Text library in the root of CIRCUITPY, they won't have access to the newest version. 

I think this should only change here after there is a new core release that includes the updated Display Text frozen in

Reply:
Portal_Base is also a frozen library. Maybe some code that checks if the Bitmap Label has `outline_color` or `outline_size` properties might work well enough.